### ISSUE TITLE
Install error for rCharts

### ISSUE THREAD

**Issue by hshans** [ramnathv/rMaps#11]
Hi,
I get this error on both 2.1.5 and 3.0.2 
package ‘rCharts’ is not available (for R version 3.0.2) 

Is there a specific version on which this will work?

Thanks,
Harry

**Comment by ffgama** [comment]
i've been the same problem @ramnathv . I did the same steps that @dicenslice but the error launched:

Installing package into ‘/Users/pma014/Library/R/3.1/library’
(as ‘lib’ is unspecified)
During startup - Warning messages:
1: Setting LC_CTYPE failed, using "C" 
2: Setting LC_TIME failed, using "C" 
3: Setting LC_MESSAGES failed, using "C" 
4: Setting LC_MONETARY failed, using "C" 
Warning: invalid package 'rCharts.tar.gz'
Error: ERROR: no packages specified
Warning in install.packages :
  installation of package ‘rCharts.tar.gz’ had non-zero exit status

:/

**Comment by PKBibi** [comment]
Hi Ramnathv

no luck either...

Here is the output I get, can you help please?
Also not able to install any git hub package with dev tools.

> install.packages("rCharts.tar.gz", repos = NULL, type = "source")
Installing package into ‘C:/Users/PK/Documents/R/win-library/3.2’
(as ‘lib’ is unspecified)
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
Warning in install.packages :
  running command '"C:/Program Files/R/R-32~1.1/bin/x64/R" CMD INSTALL -l "C:\Users\PK\Documents\R\win-library\3.2" "rCharts.tar.gz"' had status 1
Warning in install.packages :
  installation of package ‘rCharts.tar.gz’ had non-zero exit status

OS = Windows 7
RStudio Version 0.99.447

**Comment by ramnathv** [comment]
Can you try this to check if you have the build tools installed on your computer?

```r
library(devtools)
has_devel()
```

**Comment by PKBibi** [comment]
On Sat, Jun 27, 2015 at 7:42 PM, Ramnath Vaidyanathan <
notifications@github.com> wrote:

> library(devtools)
> has_devel()
>
>
Hi Ramnath

Thanks for your rapid response.

Here is the output when I run the 2 commands:

> library("devtools", lib.loc="~/R/win-library/3.2")
> has_devel()
"C:/Program Files/R/R-32~1.1/bin/x64/R" --no-site-file --no-environ
--no-save --no-restore CMD  \
  SHLIB foo.c

'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
Error: Command failed (1)
>

Not sure what it all means....
-- 
Kind regards

P. Bibi

**Comment by ramnathv** [comment]
Okay. That explains it. Follow the instructions in this thread on [SO](http://stackoverflow.com/questions/20963942/problems-with-install-bitbucket). In short, you need to uninstall R and reinstall it in a root folder like `C:\`.

**Comment by PKBibi** [comment]
Yes !

Ramnath you are the man... :)

worked like a charm.

Very much appreciated, thanks for your time and expertise.

Let's keep in touch.
https://uk.*linkedin*.com/in/pkentabibi


On Sat, Jun 27, 2015 at 8:00 PM, Ramnath Vaidyanathan <
notifications@github.com> wrote:

> Okay. That explains it. Follow the instructions in this thread on SO
> <http://stackoverflow.com/questions/20963942/problems-with-install-bitbucket>.
> In short, you need to uninstall R and reinstall it in a root folder like
> C:\.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ramnathv/rMaps/issues/11#issuecomment-116120567>.
>



-- 
Kind regards

P. Bibi

**Comment by ramnathv** [comment]
Glad to know it worked!

**Comment by Alsaxian** [comment]
Hi dear ramnathv,

I also got an error message while following your instructions
```r
require(devtools) 
install_github('rCharts', 'ramnathv')
```
to install rCharts which says
> ** installing *source* package 'rCharts' ...
> ** R
> ** demo
> ** inst
> ** preparing package for lazy loading
> ** help
> *** installing help indices
> ** building package indices
> ** testing if installed package can be loaded
> *** arch - i386
> Warning in library(pkg_name, lib.loc = lib, character.only = TRUE, logical.return = TRUE) :
>   there is no package called 'rCharts'
> Erreur : le chargement a échoué
> Exécution arrêtée

The last means loading failed and execution stopped.

The `has_devel()` command also reports an error

> "C:/PROGRA\~1/R/R-33\~1.0/bin/x64/R" --no-site-file --no-environ --no-save --no-restore --quiet CMD SHLIB foo.c 
Warning message:
l'exécution de la commande 'make -f "C:/PROGRA\~1/R/R-33\~1.0/etc/x64/Makeconf" -f "C:/PROGRA\~1/R/R-33\~1.0/share/make/winshlib.mk" SHLIB="foo.dll" WIN=64 TCLBIN=64 OBJECTS="foo.o"' renvoie un statut 127 
Error: Command failed (1)

According to what I understand from the previous comments if I'am working on a school terminal and don't have the admin permission to reinstall anything then I can't do much about it?

Thank you in advance,

Alsaxian

**Comment by GuoweiYang19891101** [comment]
Hi dear ramnathv,

I have tried this code to install rCharts:

require(devtools)
install_github('rCharts', 'ramnathv')

It returned me this error:

Error in parse_repo_spec(repo) : 
  Invalid git repo specification: 'rCharts'

Did you know why this happens?

Thank you.

Guowei Yang

**Comment by Alsaxian** [comment]
这位大神，为什么抄送给我？

Guowei Yang <notifications@github.com> 于 2019年2月25日周一 下午6:42写道：

**Comment by swapna9481** [comment]
package is not available (for R version 3.5.2)
when i try to install rchart package. but it seems warning message package is not available.

**Comment by idoiaguirre** [comment]
Hello everyone,
I had the same problem as stated above and I was able to solve it by installing 
the "devtools" package and installing "rCharts" by the "install.github" command: 

```
library("devtools")
install_github("ramnathv/rCharts")
```

Idoia Aguirre
